src/packages/views.py

```diff
@@ -133,3 +199,20 @@ def staff_view(request):
     }
 
     return render(request, 'packages/staff.html', context)
+
+
+def get_address(city_name, area, street, street_number):
+    city = get_city(city_name, area)
+    address = Address.objects.filter(city=city, street_name=street, street_number=street_number)

```

Comment:
Take a look at [`get_or_create`](https://www.kite.com/python/docs/django.db.models.QuerySet.get_or_create)

Reply:
Forgot it's a thing...

Reply:
Yet you're not using it still?

Reply:
I didn't realize it was an actual method in query set, my bad